packages/flight-icons/scripts/build-parts/generateBundleSVGReact.ts

```diff
@@ -14,7 +14,7 @@ import { ConfigData } from '../@types/ConfigData';
 import { AssetsCatalog } from '../@types/AssetsCatalog';
 import { getCssForIconAnimation } from './getCssForIconAnimation';
 
-const prettierConfig = { parser: 'typescript' as const, tabWidth: 4, singleQuote: true };
+const prettierConfig = { parser: 'typescript' as const, tabWidth: 4, singleQuote: true, trailingComma: 'none' as const };
```

Comment:
should the `as const` go outside of the `{ ... }` block? 🤔

Reply:
Yes, good catch! Placing on outside will strongly type all options. Pushed update.